src/reducers/bookReducer.js

```diff
@@ -0,0 +1,40 @@
+import { CREATE_BOOK, REMOVE_BOOK } from '../actions/index';
+import randomNumber from '../logic';
+
+const initialState = {
```

Comment:
let use `combineReducers` to combine our `bookReducer.js`, for now, our combineReducer will only take in just the  `bookReducer.js`
![image](https://user-images.githubusercontent.com/31897434/74969031-2b454980-541c-11ea-8818-b3ed6c60fc12.png)
https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/44896827/